### DISCUSSION TITLE
v0.1.0-alpha.4

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#23] [Releases]
## What's Changed
* GDPR ack option by @alexdelprete in https://github.com/alexjustesen/speedtest-tracker/pull/21

## New Contributors
* @alexdelprete made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/21

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha.3...v0.1.0-alpha.4

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.1.0-alpha.4'>v0.1.0-alpha.4</a>.</em>

**Comment by alexdelprete** [comment]
Good job. It's working. :)

I didn't see anything in the logs...then I checked the db directly and I found the data in the results table.

Couple of things:

- when a task is completed, there's no evidence in the log? can we set the log level in some way in the config.yml file?
- in the config.yml file, the `ookla_server_id` entry, does it accept multiple servers? 

Thanks a lot Alex. :)

**Reply by alexjustesen** [reply]
No multiple servers yet, and I really like the log idea. I'll get that in the pipeline. Just started working on some UI stuff

**Reply by Nexxed** [reply]
> Just started working on some UI stuff

How are you planning on doing the UI?
I'd love to get involved with this project and spice-up the UI, as I wasn't a huge fan of the way the previous project was designed and laid-out. I'm familiar with Tailwind, React and Vue if it helps! 👍

**Reply by alexdelprete** [reply]
The old project's UI was really the weak point, I hope you guys can make a beautiful and dynamic UI with all bells&whistles. :)

**Reply by alexjustesen** [reply]
@Nexxed I was thinking of going full admin panel and using https://filamentphp.com/. Built in charting and would making creating pages to manage settings pretty easy.

I was going to try and avoid abstracting the frontend to a vue/react just to keep things simple.

**Reply by alexjustesen** [reply]
@Nexxed I'm going to add onto this though, could definitely use help mocking up the UI and any design work around that if it falls into your wheelhouse

**Reply by Nexxed** [reply]
I would help with that but I haven't touched Laravel since 2019 and my PHP is very rusty, so I would likely be more of a detriment to the project than anything.

What I had in mind for this was using Laravel as the backend (as well as an API) and building the frontend using Next.js, Tailwind, and ApexCharts. If this was the path you'd be willing to take with this, we could have the frontend query the API to get an array of settings and go from there.

The UI doesn't need to be an overkill admin panel to be honest, it could be something simple and minimal but also powerful for the use case.

**Reply by alexdelprete** [reply]
> The UI doesn't need to be an overkill admin panel to be honest, it could be something simple and minimal but also powerful for the use case.

The filamentphp [demo](https://demo.filamentphp.com/) looks exactly like the type of UI we need for this project: simple, minimal, like you said.

The idea of having a flexible backend API with a modern frontend is really nice. But also the idea of having a sort of framework for the UI that is easy to maintain/manage is also nice.

Hope you can find a way to collaborate...:)

**Reply by Nexxed** [reply]
I agree, Filament does look like a good candidate for it and that demo is very functional.
I'll look into it and try to get myself familiar so I can collaborate with you and others!

**Reply by alexdelprete** [reply]
> I'll look into it and try to get myself familiar so I can collaborate with you and others!

That is great news! I'm confident this project will be awesome, you two seem like very good professionals. I'm an old dev (moved to mmgmt roles in IT and don't code since many many years) but I know the foundations and enjoy collaborating on projects with people like you two. Count me in for ideas, testing, brainstorming, solving issues, supporting the project in general. :)

Thank you so much guys.

**Reply by alexjustesen** [reply]
@alexdelprete @Nexxed Filament admin panel has been included in the latest build, let me know your thoughts in that discussion.

**Reply by alexdelprete** [reply]
> @alexdelprete @Nexxed Filament admin panel has been included in the latest build, let me know your thoughts in that discussion.

Great news, thanks. I replied in the other thread.

**Comment by alexdelprete** [comment]
for the multiple servers: the old ST project did a nice round-robin of the list of the servers, I think it was a nice feature.

glad you liked the log idea...it's always nice to know what's going on inside the container. :)
